misago/users/management/commands/removeoldips.py

```diff
@@ -3,7 +3,7 @@
 
 from misago.conf import settings
 from misago.core.utils import ANONYMOUS_IP
-from misago.users.signals import anonymize_old_ips
+from misago.users.signals import remove_old_ips
 
 class Command(BaseCommand):
     help =  "Anonymizes users IPs stored for longer than set in MISAGO_IP_STORE_TIME."
```

Comment:
Change this line to `help = "Removes users IPs stored for longer than set in MISAGO_IP_STORE_TIME."`